packages/dropdowns/src/styled/items/StyledNextItem.ts

```diff
@@ -5,23 +5,26 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import classNames from 'classnames';
 import styled from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import MenuStyles from '@zendeskgarden/css-menus';
 
-import { StyledItem } from '../';
+import { StyledItem } from './StyledItem';
+import { StyledItemIcon } from './StyledItemIcon';
 
 const COMPONENT_ID = 'dropdowns.next_item';
 
 /**
  * Accepts all `<li>` props
  */
-export const StyledNextItem = styled(StyledItem).attrs(props => ({
+export const StyledNextItem = styled(StyledItem).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(props.className, MenuStyles['c-menu__item--next'])
-}))`
+  'data-garden-version': PACKAGE_VERSION
+})`
+  ${StyledItemIcon} {
+    right: ${props => (props.theme.rtl ? 'auto' : '0')};
+    left: ${props => (props.theme.rtl ? '0' : 'auto')};
+  }
```

Comment:
Is there a particular reason why these styles were declared here, rather than directly within `StyledItemIcon`?

Reply:
The particular R/L icon positioning is only relevant for a `NextItem`.